specification/app/resource-manager/Microsoft.App/stable/2024-03-01/examples/Job_GetDetector.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Nit: Naming of examples as <resource>-<action/verb>-<subresource> i.e. Job_Get_Detector.json.

I prefer `DetectorByJob_Get.json`. Refer https://github.com/Azure/azure-openapi-validator/blob/main/docs/operation-id-noun-verb.md